tests/mysql_delete/mysql_delete.bal

```diff
@@ -0,0 +1,45 @@
+import ballerina/http;
+import ballerina/log;
+import ballerinax/mysql;
+import ballerinax/mysql.driver as _;
+import ballerina/sql;
+
+configurable string host = ?;
+configurable string username = ?;
+configurable string password = ?;
+configurable string database_name = ?;
+configurable int port = ?;
+configurable string table_name = ?;
+int id = 1;
+int count = 0;
+
+mysql:Client dbClient = check new (host = host, user = username, password = password);
+
+service /db on new http:Listener(9092) {
+    resource function post .(http:Caller caller, http:Request clientRequest) {
```

Comment:
Shall we use the correct HTTP Delete method for delete operation. and also we don't need caller objects, instead, we can just return the response and error.

Reply:
and pass the id as query param